src/w3c/templates/cgbg-sotd.js

```diff
@@ -4,32 +4,37 @@ export default (conf, opts) => {
   const html = hyperHTML;
   return html`
     <h2>${conf.l10n.sotd}</h2>
-    ${conf.isPreview
-      ? html`
-          <details class="annoying-warning" open="">
-            <summary
-              >This is a
-              preview${conf.prUrl && conf.prNumber
-                ? html`
-                    of pull request
-                    <a href="${conf.prUrl}">#${conf.prNumber}</a>
-                  `
-                : ""}</summary
-            >
-            <p>
-              Do not attempt to implement this version of the specification. Do
-              not reference this version as authoritative in any way.
-              ${conf.edDraftURI
-                ? html`
-                    Instead, see
-                    <a href="${conf.edDraftURI}">${conf.edDraftURI}</a> for the
-                    Editor's draft.
-                  `
-                : ""}
-            </p>
-          </details>
-        `
-      : ""}
+    ${
+      conf.isPreview
```

Comment:
@saschanaz do you recall why ESLint is doing this with the whitespace? I can recall how to fix this :( (Visual Studio Code seems to do the right thing)

Reply:
can't recall*

Reply:
prettier/prettier#5771 introduced this behavior. Maybe we can `npm run lint -- --fix` again and things will be fixed.